packages/ti_stix/data_stream/indicator/elasticsearch/ingest_pipeline/indicator-file.yml

```
@@ -0,0 +1,59 @@
+---
+description: Pipeline for processing file STIX patterns
+processors:
+  - grok:
+      field: _ingest._value
+      tag: grok_file_pattern
+      patterns:
+        - "^\\[?file:hashes.md5%{SPACE}=%{SPACE}'%{DATA:_tmp.md5}'\\]?"
+        - "^\\[?file:hashes.MD5%{SPACE}=%{SPACE}'%{DATA:_tmp.md5}'\\]?"
+        - "^\\[?file:hashes.'md5'%{SPACE}=%{SPACE}'%{DATA:_tmp.md5}'\\]?"
+        - "^\\[?file:hashes.'MD5'%{SPACE}=%{SPACE}'%{DATA:_tmp.md5}'\\]?"
+        - "^\\[?file:hashes.sha1%{SPACE}=%{SPACE}'%{DATA:_tmp.sha1}'\\]?"
+        - "^\\[?file:hashes.SHA1%{SPACE}=%{SPACE}'%{DATA:_tmp.sha1}'\\]?"
+        - "^\\[?file:hashes.'sha-1'%{SPACE}=%{SPACE}'%{DATA:_tmp.sha1}'\\]?"
+        - "^\\[?file:hashes.'SHA-1'%{SPACE}=%{SPACE}'%{DATA:_tmp.sha1}'\\]?"
```

```suggestion
        - "(?i:^\\[?file:hashes.sha-?1%{SPACE}=%{SPACE}'%{DATA:_tmp.sha1}'\\]?)"
```
Should `hashes.sha` be `hashes\.sha`?